### PR TITLE
Issue with infinite spinner - correct useNetwork

### PR DESCRIPTION
**Pull request by jmzwar**

### CODE REVIEW

File: v3/lib/usePools/usePools.ts [hunk 1]
```diff
@@ -26,12 +29,16 @@ const loadPoolNames = async ({
...
 };
+
 export const usePools = () => {
+  const network = useNetwork();
   const { data: CoreProxyContract } = useCoreProxy();
   const { data: preferredPool } = usePreferredPool();
+
   return useQuery({
-    queryKey: [{ CoreProxyContract: CoreProxyContract?.address, preferredPool }, 'pools'],
+    queryKey: [network.name, 'pools'],
```
Comment (bachstatter) [inline v3/lib/usePools/usePools.ts:39]: In theory preferredPool pool could change

File: v3/lib/useCollateralTypes/useCollateralTypes.ts [hunk 2]
```diff
@@ -94,17 +98,16 @@ async function loadCollateralTypes({
...
+
   return useQuery({
-    queryKey: [
-      { Multicall: MulticallContract?.address, CoreProxy: CoreProxyContract?.address },
-      'collateralTypes',
-    ],
+    queryKey: [network.name, 'collateralTypes'],
     queryFn: async () => {
-      if (!CoreProxyContract || !MulticallContract) {
+      if (!network.isSupported) throw new Error('Unsupported Network');
```
[the remark this answers is not in the document]
Comment (bachstatter) [reply v3/lib/useCollateralTypes/useCollateralTypes.ts:108]: Yea I think this make sense

[the remark this answers is not in the document]
Comment (jmzwar) [reply v3/lib/useCollateralTypes/useCollateralTypes.ts:108]: We can, but I'd want to see how we approach error pages first.

File: v3/lib/usePools/usePools.ts (hunk 1 above)
Comment (jmzwar) [reply v3/lib/usePools/usePools.ts:39]: Ok - will add it back.

File: v3/lib/useCollateralTypes/useCollateralTypes.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (jmzwar) [reply v3/lib/useCollateralTypes/useCollateralTypes.ts:108]: Yeah - what I'll do is remove them for now. We can always re-add them later.

**Review by bachstatter — APPROVED** [review]

**Pull request merged**